gsms.py

```diff
@@ -180,12 +184,36 @@ def main() -> None:
 
                 if not app_exists:
                     count += 1
+                    if shortcut.work_dir.endswith(os.sep): # remove final path separator but only if it exists
+                       shortcut.work_dir = shortcut.work_dir[:-1]
+
+                    target_path = shortcut.path
+
+                    # prepare regex to get folder UUIDs
+                    regex = re.compile(r"^::(\{[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\})\\")
+
+                    work_dir_result = regex.findall(shortcut.work_dir)
+
+                    if len(work_dir_result) == 1:
+                        shortcut.work_dir = shortcut.work_dir.replace(f"::{work_dir_result[0]}", get_win_path(work_dir_result[0]))
+
+                    path_result = regex.findall(shortcut.path)
+
+                    if len(path_result) == 1:
+                        target_path = shortcut.path.replace(f"::{path_result[0]}", get_win_path(path_result[0]))
+
+                    target_path = target_path.replace(shortcut.work_dir, '')
+
+
+                    if target_path.startswith(os.sep) and os.name == "nt": # remove first path separator but only if it exists
```

Comment:
comment should have 2 spaces before or be on the line above (linter will complain)

Reply:
moved above